src/marqo/s2_inference/model_registry.py

```diff
@@ -817,6 +953,14 @@ def _get_hf_properties() -> Dict:
                  "poolingMethod": "mean",
                  "text_query_prefix": "Represent this sentence for searching relevant passages: ",
                  "notes": ""},
+            "BAAI/bge-base-en-v1.5":
+                {"name": 'BAAI/bge-base-en-v1.5',
+                 "dimensions": 768,
+                 "tokens": 512,
+                 "type": "sbert",
+                 "poolingMethod": "mean",
```

Comment:
This should be `cls`. Check [here](https://huggingface.co/BAAI/bge-base-en-v1.5/blob/main/1_Pooling/config.json#L3). This comment also applied to other models.

Reply:
discussed offline